src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -155,6 +155,18 @@ public static void showFirstBlacklistedPersonOnly(Model model) {
     }
 
     /**
+     * Updates {@code model}'s filtered whitelist to show only the first person in the {@code model}'s address book.
+     */
+    public static void showFirstWhitelistedPersonOnly(Model model) {
+        ReadOnlyPerson person = model.getAddressBook().getWhitelistedPersonList().get(0);
+        final String[] splitName = person.getName().fullName.split(ONE_OR_MORE_SPACES_REGEX);
+        model.updateFilteredWhitelistedPersonList(new NameContainsKeywordsPredicate(Arrays.asList(splitName[0])));
+
+        assert model.getFilteredWhitelistedPersonList().size() == 1;
+    }
+
```

Comment:
try to leave at most one blank line between methods and logical blocks

Reply:
Noted.